server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -547,6 +548,8 @@ private Modal blockDescriptionModal(Tag csrfTag, BlockForm blockForm, String blo
         .withId("block-edit-form")
         .with(
             div(
+                    h3("The following fields will only be visible to administrators")
```

Comment:
Did you consider changing it from "screen name" to something with a little less assumptions associated with it?  Like "Label" or "Nickname" ?

Reply:
@rahaghas was considering revamping the language around screens / blocks. We should likely do that as part of a more holistic change though. "Name / Description" are consistent with the terminology we use for programs and questions, so introducing separate language here would be a bit inconsistent. I'm definitely open to switching the language we use within the Admin UI, I just think we should do it as one atomic change. Filed #2668 for that.

Reply:
Thanks!